ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,245 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Track the workflow process
+
+A process is an instance of a workflow. When a transaction is performed, a piece of data is generated, for example an invoice, which in turn initiates a workflow process. Therefore, for every single transaction performed, an individual instance of the corresponding workflow, that is, a process, is initiated.
+
+Similarly to a workflow, a process has a start and an end point, as well as different stages (places and actions) which the initial data goes through in order to be processed. For a detailed description of the workflow elements, see [Workflow elements](#01_WorkflowElements.md).
+
+The workflow process can be tracked to check the status of a whole process or of individual actions, and to solve any problems that may occurred as a result of a misconfiguration or any other external issues.
+
+
+## Check the process status
+
+The *OVERVIEW* tab in the *Processes* menu entry displays a list of all processes that are currently being executed or have been executed in the system. Per default, all existing processes are displayed but can also be filtered according to different criteria.
+
+[comment]: <> (UI info -> columns, filter, etc.)
+
+A *VIEW* can also be created to customize the display options. For detailed information, see [Link](#to-be-completed.).
+
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated, see [Link](#to-be-determined).
+
+### Procedure
+
+*Workflows > Menu entry Processes > Tab OVERVIEW*
+
+![Processes](/Assets/Screenshots/ActindoWorkFlow/Workflows/Processes.png "[Processes]")
+
+1. Click *Status* drop-down list to filter the processes according to their status, if necessary.   
+All existing processes with the selected status are displayed.
+
+  The following statuses are available:
+
+  - **All**  
```

Comment:
Darstellung Optionen: 
- **All** : All existing processes are displayed. This option is selected per default.